web/server/codechecker_server/migrations/config/versions/7ed50f8b3fb8_new_table_for_personal_access_tokens.py

```diff
@@ -50,25 +43,6 @@ def upgrade():
             'user_name',
             'token_name', name=op.f('uq_personal_access_tokens_user_name'))
     )
-
-    one_year_later = datetime.now() + timedelta(days=365)
-
-    dialect = op.get_context().dialect.name
-
-    if dialect == "sqlite":
-        random_string = "hex(randomblob(4))"
-    else:
-        random_string = "substr(md5(random()::text), 1, 8)"
-
-    op.execute(
-        f"""
-        INSERT INTO personal_access_tokens (user_name, token_name, token,
-            description, last_access, expiration, auth_session_id)
-        SELECT user_name, {random_string}, token, description, last_access,
-            '{one_year_later}', id
-        FROM auth_sessions
-        WHERE can_expire = false
-        """)
```

Comment:
The personal access tokens should still be migrated from the `auth_sessions` table. We wouldn't like to lose the already generated personal access tokens.